setup.py

```diff
@@ -54,14 +53,29 @@ def read_file(fname):
 static_requirements = []
 static_dir = os.path.dirname(os.path.realpath(kolibri_dist.__file__))
 
-install_requires = [str(ir.req) for ir in parse_requirements('./requirements.txt', session=False)]
+install_requires = [
+    'cherrypy==6.0.2',
+    'django-filter==0.13.0',
+    'django-js-reverse==0.7.2',
+    'six==1.10.0',
+    'djangorestframework==3.3.3',
+    'django==1.9.7',
+    'colorlog==2.6.0',
+    'docopt==0.6.2',
+    'django-mptt>=0.8.4',              # apparently you also need to have the dependency_links packages be included in install_requires
+]
+
+dependency_links = [
+    'http://github.com/django-mptt/django-mptt/tarball/9e131f91#egg=django-mptt-0.8.4'
+]
 
 # Check if user supplied the special '--static' option
 if '--static' in sys.argv:
     sys.argv.remove('--static')
     dist_name = 'kolibri-static'
     description += " This static version bundles all dependencies."
-    install_requires, static_requirements = [], install_requires
+    install_requires, static_requirements = [], (install_requires + dependency_links)
+    dependency_links = []
```

Comment:
Actually after testing, no. For UNICEF, since we can't depend on them having Git, we'll have to include a `django-mptt` tarball in our Windows Installer, and then insert that in our `dependency_links` var.